### PR TITLE
Missing `f` prefix on f-strings fix

### LINKED ISSUE #7137: Missing `f` prefix on f-strings
Some strings looks like they're meant to be f-strings but are missing the `f` prefix meaning variable interpolation won't happen.

https://github.com/qutebrowser/qutebrowser/blob/07ce0349837fffc221184857be56517a2d208bd8/qutebrowser/misc/sql.py#L486
https://github.com/qutebrowser/qutebrowser/blob/07ce0349837fffc221184857be56517a2d208bd8/qutebrowser/browser/webkit/http.py#L162
https://github.com/qutebrowser/qutebrowser/blob/07ce0349837fffc221184857be56517a2d208bd8/scripts/dev/change_release.py#L59
https://github.com/qutebrowser/qutebrowser/blob/07ce0349837fffc221184857be56517a2d208bd8/scripts/dev/change_release.py#L61

I found this issue automatically. I'm a bot. Beep Boop 🦊. See other issues I found in your repo [here](https://codereview.doctor/qutebrowser/qutebrowser)

### PR DESCRIPTION
**Pull request by code-review-doctor**
Fixes #7137

### CODE REVIEW

File: qutebrowser/misc/sql.py [hunk 1]
```diff
@@ -483,7 +483,7 @@ def delete(self, field: str, value: Any) -> None:
         q = self.database.query(f"DELETE FROM {self._name} where {field} = :val")
         q.run(val=value)
         if not q.rows_affected():
-            raise KeyError('No row with {field} = "{value}"')
+            raise KeyError(f'No row with {field} = "{value}"')
```
Comment (emanuele6) [inline qutebrowser/misc/sql.py:486]: I know this is a bot, but it probably makes more sense to also use `{value!r}` instead of `"{value}"` here

Comment (code-review-doctor) [reply qutebrowser/misc/sql.py:486]: I'm half bot half human.

Our bot detected the problem, then a real developer triaged the issue and approved the bot to create the issue and PR. 

Now it's back in the hands of the humans monitoring the PRs :)

I'll make that change now

Comment (emanuele6) [reply qutebrowser/misc/sql.py]: > I'll make that change now

You didn't remove the quotes.

**Comment by The-Compiler** [comment]
I was curious on how this happened, because I remember much of that code to be much older than f-strings... Well, turns out I'm apparently not very good in rewriting f-strings by hand :see_no_evil:.

- bc64ea7fad9be0e75e13ece0ec20e07f04736fb3 ("Improve error handling", yeah...)
- dfed3cecaf2fe27776a419d8e11dd558fb08758f
- 6adf86ec496babac580b88c8baacf6b2710d8ca1 (new code, funnily enough using a mixture of `.format` and not-quite-fstrings, apparently)

One more reason I'm really looking forward to finally rewriting them all automatically via #1455 soon.

@emanuele6 was spot on re the redundant quotes now that `!r` is used:

```python
>>> field = "name"
>>> value = "test"
>>> print(f'No row with {field} = "{value}"')
No row with name = "test"
>>> print(f'No row with {field} = "{value!r}"')
No row with name = "'test'"
>>> print(f'No row with {field} = {value!r}')
No row with name = 'test'
```

Fixed that up after the merge.

Thanks @code-review-doctor and @emanuele6! :sparkles: 

---

Also, I have to say this is quite impressive, @code-review-doctor! Last time I tried the website, it mostly gave me things I wasn't interested in (would be cool to have a "hide this category of errors" toggle, even just for the demo). Given how other linters have been really struggling with detecting missing f-string prefixes (see e.g. https://github.com/PyCQA/pylint/issues/2507), this is great work (assuming you didn't have to filter through false-positives manually before opening this).

I still don't like relying on online-only services for linting (prefer something I can just run locally and integrate into GitHub Actions without relying on third-party stuff), and I'm surprised there is still room for another such service (Deepsource, Sonarqube, Codacy, Codeclimate, etc. etc. etc.), but you seem to be doing a great job ([see also](https://blog.devgenius.io/5-of-666-python-repos-had-comma-typos-including-tensorflow-and-pytorch-sentry-and-v8-7bc3ad9a1bb7?gi=877a6b9e3a90))!

**Pull request merged**